doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -52,6 +52,27 @@ Build system
 
   * Manifest file entry ``mbedtls`` (:makevar:`ZEPHYR_MBEDTLS_MODULE_DIR`) checked out at path :file:`modules/crypto/mbedtls` now points to |NCS|'s fork of MbedTLS instead of Zephyr's fork.
 
+Board support
+-------------
+
+* nRF52 and nRF53 series development kits:
+
+  * Added :kconfig:option:`CONFIG_NCS_SAMPLE_MCUMGR_BT_OTA_DFU` and :kconfig:option:`CONFIG_NCS_SAMPLE_MCUMGR_BT_OTA_DFU_SPEEDUP` to configure FOTA updates over Bluetooth Low Energy in the default setup.
+    The default setup uses MCUmgr libraries with the Bluetooth transport layer and requires the user to enable MCUboot bootloader.
+    See details in :ref:`ug_nrf52_developing_ble_fota` section of development guide.
```

Comment:
If there is distinct UG FOTA ref for nRF53 DK, consider updating this part

```suggestion
    See details in :ref:`ug_nrf52_developing_ble_fota` or (link to nrf53 UG section with fota) section of development guide.
```

Reply:
In nrf5340.rst there is such snippet:
```
.. include:: ../nrf52/developing.rst
   :start-after: fota_upgrades_start
   :end-before: fota_upgrades_end
```
so in a nutshell it should be the same.

Reply:
But as i can see in CI there is something wrong with that :ref:`ug_nrf52_developing_ble_fota` so technical writers needs to take a look at this anyway

Reply:
So, I edited Kamil's suggestion and it should work.

Reply:
I've missed that one, pushed change right now: https://github.com/nrfconnect/sdk-nrf/compare/8924e7e3676114a5192eed70f6ee812431b5a61d..0344a073b6af2b7bfaedcf3286d594a29e5d99c5